selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CheMultiuserAdminDashboard.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.pageobject.dashboard;
+
+import static java.util.Collections.singletonList;
+import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC;
+import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.REDRAW_UI_ELEMENTS_TIMEOUT_SEC;
+import static org.openqa.selenium.support.ui.ExpectedConditions.invisibilityOfAllElements;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOf;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.eclipse.che.selenium.core.provider.TestDashboardUrlProvider;
+import org.eclipse.che.selenium.core.provider.TestIdeUrlProvider;
+import org.eclipse.che.selenium.core.user.TestUser;
+import org.eclipse.che.selenium.pageobject.site.LoginPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Page object extends {@link Dashboard} page object to cover elements which are specific for
+ * Codenvy Admin dashboard content.
+ *
+ * @author Dmytro Nochevnov
+ */
+@Singleton
+public class CheMultiuserAdminDashboard extends Dashboard {
+
+  @Inject
+  public CheMultiuserAdminDashboard(
+      SeleniumWebDriver seleniumWebDriver,
+      TestUser defaultUser,
+      TestIdeUrlProvider testIdeUrlProvider,
+      TestDashboardUrlProvider testDashboardUrlProvider,
+      LoginPage loginPage) {
+    super(seleniumWebDriver, defaultUser, testIdeUrlProvider, testDashboardUrlProvider, loginPage);
+    PageFactory.initElements(seleniumWebDriver, this);
+  }
+
+  private interface Locators {
+    String SETTINGS_ITEM_XPATH = "//a[@href='#/onprem/administration']";
```

Comment:
Did you check on this path? It's strange to see "**onprem**" word as part of it.

Reply:
Tests are not shown any problems related with this xpath. So this locator is actual or it does not used in the tests.

Reply:
It's a link to the administration page inside Dashboard. Could you, please, verify that the link is actual, and if so, ask Dashboard team to change it to have a proper name?